example/helm/values.yaml

```diff
@@ -63,15 +64,15 @@ agent:
       memory: 150Mi
   ngt:
     # The number of dimensions for feature vector of fashion-mnist dataset.
-    dimension: 784
+    dimension: 300
     # We use L2-Norm for distance_type.
-    distance_type: l2
+    distance_type: cos
     # The type of fashion-mnist's feature vectors.
     object_type: float
     # Check duration of automatic indexing.
-    auto_index_check_duration_limit: 1m
+    auto_index_check_duration_limit: "-1m"
```

Comment:
what is "-1m" means here?
if set to minus value i recommend "-1" to avoid confusion.

Reply:
not working.

Reply:
It might be better to set it to "0s" to completely disable the daemon.
Ref: https://github.com/vdaas/vald/blob/main/pkg/agent/core/ngt/service/ngt.go#L201-L203
https://github.com/vdaas/vald/blob/main/pkg/agent/core/ngt/service/ngt.go#L503-L505

Reply:
Also I think it is better to add a new configuration like "AutoIndexDisabled" to allow user to easier to disable this feature instead of setting this confusing value (IMO).

Reply:
It might be another discussion/enhancement of Vald. I will approve this PR first.